src/views/Proposals.vue

```diff
@@ -115,12 +117,15 @@ export default {
           .sort((a, b) => b[1].msg.payload.end - a[1].msg.payload.end, 0)
       );
     },
-    isMember() {
-      const members = this.space.members.map(address => address.toLowerCase());
+    isOwner() {
+      return this.spaceUri.includes(this.web3.account);
+    },
+    isAdmin() {
+      const admins = this.space.admins.map(address => address.toLowerCase());
```

Comment:
@samuveth Getting error if no admins field in space setting
![image](https://user-images.githubusercontent.com/15967809/117331366-4fbc0300-aeb4-11eb-953d-ce12b427d8c0.png)

Reply:
Hm, I wonder why it works for `isMember` like that.

Reply:
If `members` is always defined then `admins` should be too no?

Reply:
If admins is always defined, then 99% of the space will fail to load, cuz the hub check if all space settings are correct. Both should be option fields.

Reply:
@bonustrack I mean like members, it's always at minimum `members: []` when the space setting are loaded from the hub. Or is that not the case?

Reply:
It's not necessary the case

Reply:
Then some spaces must be getting an error here: https://github.com/snapshot-labs/snapshot/blob/7b6d7c80d408596dda5f0192e4286b9a0018246d/src/views/Proposals.vue#L115

Reply:
Not with that https://github.com/snapshot-labs/snapshot/blob/a53474ba13edd8ffbf480c71ace3e2c7b8da885f/src/helpers/utils.ts#L87

Reply:
That's what I was looking for!